lib/kvdb/kvdb_rparams.c

```diff
@@ -795,23 +795,23 @@ static const struct param_spec pspecs[] = {
         },
     },
     {
-        .ps_name = "csched_vb_scatter_pct",
-        .ps_description = "csched vblock scatter pct. in leaves",
+        .ps_name = "csched_max_vgroups",
+        .ps_description = "maximum number of vgroups in a single kvset",
         .ps_flags = PARAM_FLAG_EXPERIMENTAL | PARAM_FLAG_WRITABLE,
         .ps_type = PARAM_TYPE_U8,
-        .ps_offset = offsetof(struct kvdb_rparams, csched_vb_scatter_pct),
-        .ps_size = PARAM_SZ(struct kvdb_rparams, csched_vb_scatter_pct),
+        .ps_offset = offsetof(struct kvdb_rparams, csched_max_vgroups),
+        .ps_size = PARAM_SZ(struct kvdb_rparams, csched_max_vgroups),
         .ps_convert = param_default_converter,
         .ps_validate = param_default_validator,
         .ps_stringify = param_default_stringify,
         .ps_jsonify = param_default_jsonify,
         .ps_default_value = {
-            .as_uscalar = 100,
+            .as_uscalar = 32,
         },
         .ps_bounds = {
             .as_uscalar = {
-                .ps_min = 0,
-                .ps_max = 100,
+                .ps_min = 1,
+                .ps_max = 128,
```

Comment:
Where does 128 come from? What keeps this value from being UINT8_MAX?

Reply:
Also, why was u8 chosen? Would be good to have some justification in a comment preceding the parameter definition in my opinion in order to justify the choices.

Reply:
u8 because the old value was a u8.  The only reason we use u8 in instead of sometthing larger is because there's no reason to every have a max value that exceeds 255.  The values 32 for default and 128 for max are guesses.  Even 32 might be too high as a default.